memo/src/processor.rs

```diff
@@ -0,0 +1,48 @@
+use solana_sdk::{
+    account_info::AccountInfo, entrypoint, entrypoint::ProgramResult, program_error::ProgramError,
+    pubkey::Pubkey,
+};
+use std::str::from_utf8;
+
+entrypoint!(process_instruction);
+fn process_instruction<'a>(
+    _program_id: &Pubkey,
+    _accounts: &'a [AccountInfo<'a>],
+    instruction_data: &[u8],
+) -> ProgramResult {
+    from_utf8(instruction_data)
```

Comment:
Seems more readable to me:

```rust
from_utf8(instruction_data).map_err(|_| ProgramError::InvalidInstructionData)?;
Ok(())
```

Maybe @mvines didn't notice the return value?

Reply:
I prefer that as well. I'll try to sneak it by him again ;)

Reply:
🙈 ok do it now!